lib/scss/components/event-filter-menu/stanford_events.event-filter-menu.scss

```diff
@@ -0,0 +1,162 @@
+@charset "UTF-8";
+@import '../../stanford_events.config';
+
+//
+// Filter menu
+//
+
+// Changing the layout for the filtered pages.
+@include grid-media-max('lg') {
+  .left-region {
+    &.flex-lg-3-of-12 {
+      flex: auto;
+      max-width: 100%;
+    }
+  }
+}
```

Comment:
@jenbreese This statement is a little broad. The `.left-region` is being adjusted everywhere it shows up on a page and we use that css class in other templates. Can you and a parent selector to scope this just to the event filter please.

Reply:
I made this more specific.

Reply:
Could you nest .left-region under another selector so it doesn't accidentally apply to some other layout when in use?